src/plone/restapi/tests/test_documentation.py

```diff
@@ -515,15 +514,41 @@ def test_documentation_types_document_crud(self):
                 "required": True,
             },
         )
-        save_request_and_response_for_docs("types_document_post_field", response)
+        # With plone.dexterity 2.10+ we get an unstable behavior name like this:
+        # plone.dexterity.schema.generated.plone_5_1630611587_2_523689_0_Document
+        # Replace it.
+        document_schema_re = re.compile(
+            r"^plone.dexterity.schema.generated.plone_.*_Document$"
+        )
+        stable_behavior = "plone.dexterity.schema.generated.plone_0_Document"
```

Comment:
I think stable_behavior should look like what it is in plone.dexterity 2.10+. That is:

plone.dexterity.schema.generated.plone_5_1630600102_2_680259_0_Document

Reply:
Technically that would be closer to what it looks like. But it would be uglier, so I prefer this clean one.

Reply:
Even being “uglier”, I prefer it closer to reality. Each part of this code has a meaning. When someone unsuspecting consults the documentation, they can get confused.